strimzi-container/src/main/java/io/strimzi/StrimziContainer.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi;
+
+import com.github.dockerjava.api.command.ExecCreateCmdResponse;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import com.github.dockerjava.api.model.ContainerNetwork;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+
+import java.util.Collection;
+import java.util.concurrent.TimeUnit;
+
+public class StrimziContainer extends GenericContainer<StrimziContainer> {
+
+    private static final Logger LOGGER = LogManager.getLogger(StrimziContainer.class);
+
+    private static final String STARTER_SCRIPT = "/testcontainers_start.sh";
+    private static final int KAFKA_PORT = 9093;
+    private static final int ZOOKEEPER_PORT = 2181;
+
+    private int kafkaExposedPort;
+    private StringBuilder advertisedListeners;
+
+    public StrimziContainer(final String version) {
+        super("strimzi/kafka:" + version);
```

Comment:
How is the version here managed? Also - assuming this is an user interface ... should it be there as strimziVersion and kafkaVersion and the tag generated out of that?